### PR TITLE
Adding user tominabox1 keyboard layouts to main branch

### PR DESCRIPTION
**Pull request by tominabox1**
## Description

Plaid and KBD6x layouts added for user tominabox1

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/plaid/keymaps/tominabox1/keymap.c [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+     * | A    |   R  |   S  |   T  |   D  |  TAB |RETURN|   H  |   N  |   E  | I    |  O   |
+     * |------+------+------+------+------+------|------+------+------+------+------+------|
+     * | Z    |   X  |   C  |   V  |   B  |   [  |   ]  |   K  |   M  |   ,  |   .  |   '  |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * | CTRL | CAD  | GUI  | BSPC | LALT | TG3  | MO3  | SHIFT| SPACE|RETURN| BSLS | UP   |
+     * `-----------------------------------------------------------------------------------'
+     */
+    
+    [0] = LAYOUT_ortho_4x12(
+        KC_Q, KC_W, KC_F, KC_P, KC_G, KC_GESC, KC_DEL,  KC_J, KC_L, KC_U,    KC_Y,   KC_SCLN, \
```
Comment (fauxpark) [inline keyboards/plaid/keymaps/tominabox1/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".

File: quantum/quantum.c [hunk 2]
```diff
@@ -1011,6 +1011,7 @@ void matrix_scan_quantum() {
   #endif
 
   matrix_scan_kb();
+
```
Comment (drashna) [inline quantum/quantum.c]: Could you remove this change? 
```suggestion
```

Comment (tominabox1) [reply quantum/quantum.c]: Yes. I will revert this in my next update.

File: keyboards/plaid/keymaps/tominabox1/keymap.c (hunk 1 above)
Comment (tominabox1) [reply keyboards/plaid/keymaps/tominabox1/keymap.c]: I will remove these as requested in the next update.

**Comment by drashna** [comment]
Also, it looks like you have 2 keyboards that are using `LAYOUT_ortho_4x12`.  If that is the case, and they are the same, you may want to use the community layout feature.  That would allow you to maintain one keymap for multiple keyboards.

https://docs.qmk.fm/#/feature_layouts

Eg, just move one of them to `/layouts/community/ortho_4x12/tominabox1/`, and remove the other.

**Comment by tominabox1** [comment]
I'm going to take these comments into consideration and open a new PR to clean up some of these issues.  I appreciate the review.

**Pull request closed without merging**